setup.py

```diff
@@ -18,7 +18,7 @@
     'django-reversion>=1.8.2,<1.9',
     'django-sortedm2m',
     'django-taggit',
-    'django>=1.6,<1.8',
+    'django>=1.6,<1.8', # rq.filter: 1.6,<1.8
```

Comment:
Is this for requires.io? I am not a fan of cluttering files with comments so some 3rd party services can work with it..